src/objects/LottoShop.js

```diff
@@ -0,0 +1,32 @@
+import { MissionUtils } from '@woowacourse/mission-utils';
+import { LOTTO } from '../lib/constants.js';
+import { Lotto } from './index.js';
+
+class LottoShop {
+  static orderLottos(purchasePrice) {
+    const lottoCount = purchasePrice / LOTTO.PRICE;
+    const lottos = this.#generateLottos(lottoCount);
+
+    return lottos;
+  }
+
+  static #generateLottos(lottoCount) {
+    const lottos = new Array(lottoCount)
+      .fill()
+      .map(() => new Lotto(this.#generateLottoNumbers()));
```

Comment:
```suggestion
    const lottos = Array.from({ length: lottoCount }, 
      () => new Lotto(this.#generateLottoNumbers()));
```

여기선 `Array.from()`을 사용하면 더 간결하게 배열을 생성할 수 있을 것 같아요!

Reply:
그 방식도 좋을 거 같습니다 ☺️